src/chrome/content/rules/All-Inkl.xml

```diff
@@ -1,23 +1,52 @@
+<!--
+	No working URL known:
+		all-inkl.es
+		www.all-inkl.es
+
+	Invalid certificate:
+		(www.)?all-inkl.ch
+		(www.)?all-inkl.de
+		(www.)?all-inkl.it
+		(www.)?all-inkl.uk
```

Comment:
@J0WI, done. I checked every single ccTLD. They don't own all of them so the previous rewrite wasn't really safe/correct.